Makefile

```diff
@@ -14,7 +14,9 @@ check:
 	cargo check
 
 test:
+	cd tests; ./setup_tests
 	cargo test
+	rm cairo_programs/*.json
```

Comment:
I think it would be more clear if  we add this removes in the setup_tests script